README.md

```diff
@@ -24,224 +29,6 @@ for specific test name of class name
 pytest -k 'TestFigureCaptionPredictor' --no-cov -n0
 ```
 
-## Quickstart guide
-
-#### 1. Create a Document for the first time from a PDF
-
-In this example, we use the `PdfPlumberParser` to convert a PDF into a bunch of text and `PDF2ImageRasterizer` to convert that same PDF into a bunch of page images.
-```python
-from typing import List
-from mmda.parsers import PDFPlumberParser
-from mmda.rasterizers import PDF2ImageRasterizer 
-from mmda.types import Document, PILImage
-
-# PDF to text
-parser = PDFPlumberParser()
-doc: Document = parser.parse(input_pdf_path='...pdf')
-
-# PDF to images
-rasterizer = PDF2ImageRasterizer()
-images: List[PILImage] = rasterizer.rasterize(input_pdf_path='...pdf', dpi=72)
-
-# attach those images to the document
-doc.annotate_images(images=images)
-```
-
-#### 2. Iterating through a Document
```

Comment:
I understand you may not want to give these same quickstart directions, but afaik this is the only documentation in the library so far of how the data model is intended to work, so it would be nice to preserve some aspects of it.

Reply:
im working on an update to this which'll patch the README